scala/zio-http/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "latest.integration")
```

Comment:
required ?

Reply:
All other frameworks have it. it's for packing it all.

Reply:
ok